app/services/payment_gateway/request.rb

```diff
@@ -1,5 +1,4 @@
 # Namespace for encapsulating Payment gateway related code
-# TODO: take a look at this rubocop warning
 # rubocop:disable Style/StructInheritance
```

Comment:
Is this still needed?

Reply:
@leoapost I didn't remove the Struct from this one so yes it's still needed.